rules.ini

```diff
@@ -148,4 +148,8 @@ Steam_Audio = (?:^|/)(?:lib)?(?:steamaudio|phonon)\.(?:dylib|dll|so)$
 Steam_Networking = (?:^|/)(?:lib)?steamnetworkingsockets\.(?:dylib|dll|so)$
 Tobii = (?:^|/)Tobii
 UnityHDRP = (?:^|/)Unity\.RenderPipelines\.HighDefinition\.(?:Config\.Runtime|Runtime)\.dll$
+UnityURP[] = (?:^|/)Unity\.RenderPipelines\.Universal\.(?:Shaders|Runtime)\.dll$
+UnityURP[] = (?:^|/)Unity\.RenderPipeline\.Universal\.(?:ShaderLibrary)\.dll$
+UnityURP[] = (?:^|/)Aragami2_Data/il2cpp_data/etc/mono$
```

Comment:
Remove this and the new line.

This is just one specific game, and we don't want to match like that. This whole method won't work on il2cpp games, and that's fine.